Documentation/community.rst

```diff
@@ -4,6 +4,67 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+Weekly Community Meeting
+========================
+
+The Cilium contributors gather every Monday at 8am PDT, 17:00 CEST, for a
+one-hour Zoom call open to everyone. During that time, we discuss:
+
+- the statuses of the next releases for each supported Cilium release
+- the current state of our CI: flakes being investigated and upcoming
+  changes
+- the development items for the next release
+- miscellaneous topics during the open session
+
+If you want to discuss something during the next meeting's open session,
+you can add it to `the meeting's Google doc
+<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+
+Zoom link: https://zoom.us/j/596609673.
```

Comment:
Are we concerned at all about bots / trawlers / trolls finding these links and causing trouble?

If so, we could state that links to the meeting notes and Zoom room are available from the #development channel in Slack. At least then it requires going through a verification step first.

Reply:
Sharing the link to the Google doc should be okay, no? I think we already share it sometimes in issues (at least I'm sure I saw it recently for the SIG-Datapath meeting notes).

Reply:
:+1: The google doc is easier to lock down if there's abuse, but we could also recreate the Zoom room if necessary.